bonfire/Modules/Recycler/Views/listResource.php

```diff
@@ -1,16 +1,17 @@
 <?php $this->extend('master') ?>
 
 <?php $this->section('main') ?>
+
 <x-page-head>
     <div class="row">
         <div class="col">
             <h2>Recycler</h2>
         </div>
         <?php if(count($resources) > 1) : ?>
             <div class="col-auto">
-                <select name="resource" class="form-select">
+                <select name="r" class="form-select" hx-get="?" hx-target="#resource" hx-boost="true">
```

Comment:
I don't have a whole lot of experience with htmx, yet, either. This is my first project using it. If this works, great! I don't know that `hx-target` is needed with `hx-boost`, though. Boost tells it to basically replace the entire page, I believe.